.github/workflows/dev_builds.yml

```diff
@@ -451,57 +451,6 @@ jobs:
             REPO_URL=sumologic/sumologic-otel-collector-dev \
             BUILD_TYPE_SUFFIX="-ubi"
 
-  package-msi:
```

Comment:
what does removing MSI and wixext do? just curious

Reply:
This is done in the packaging repository. We'll no longer be building or testing MSI or MSI code in this repository.